site/docs/master/getting-started.md

```diff
@@ -0,0 +1,49 @@
+# Getting Started
+
+Use this doc to kick the tires and get started with Velero.
+Refer [this document](advanced-installation.md) for more advanced install instructions
```

Comment:
similar to the comment on the "advanced installation" title, I'm wondering if it would be clearer to describe this as ways to customize your install, rather than calling it "advanced"

Reply:
How do you feel about `Basic Install` and `Customize Install`.

Reply:
Agree, something like "for alternative install instructions" would be more reflective of the content.

Reply:
Or, "for customized install instructions"

Reply:
Looks like this comment came in after I pushed the latest changes.
I chose
```
Refer [this document](customize-installation.md) to customize your installation.
```